test/utils/ui.test.helper.ts

```diff
@@ -40,3 +43,37 @@ export const verifyElementExistOnPage = async (
   page: puppeteer.Page,
   selector: string
 ) => !!(await page.$(selector));
+
+export const acceptCookies = async (page: puppeteer.Page) => {
+  const selector = '#rcc-confirm-button';
+  await page.waitForSelector(selector, { visible: true });
+  await page.click(selector);
+  await page.waitForSelector(selector, { hidden: true });
+};
+
+export const waitForLoadingIndicatorToHide = async (
+  page: puppeteer.Page,
+  param: boolean
```

Comment:
is this argument necessary? Either way, the name should be more meaningful.

Reply:
Will be addressed as part of the next issue.